README.md

```diff
@@ -138,6 +139,7 @@ Each distance corresponds to a distance type. The type name and the correspondin
 |  CorrDist            |  corr_dist(x, y)         | cosine_dist(x - mean(x), y - mean(y)) |
 |  ChiSqDist           |  chisq_dist(x, y)        | sum((x - y).^2 / (x + y)) |
 |  KLDivergence        |  kl_divergence(x, y)     | sum(p .* log(p ./ q)) |
+| RenyiDivergence   | renyi_divergence(x, y, k) | log(sum( x .* (x ./ y) .^ (k - 1))) / (k - 1) |
```

Comment:
Can you make the `|`s line up? Not that it actually matters, it's just bugging me. 😛